README.md

```diff
@@ -1,2 +1,7 @@
 # Red Hat Icons
 Official Red Hat Brand Icons
+
```

Comment:
Add:

  * Link to https://red-hat-icons.netlify.app/
  * How to use / "Quick Start"
    * Copy -> Paste or `npm i` into your project or ??
  * Link to contributing.md "# Contribute, Looking to contribute, see our [contribution instructions](#)."
  * Link to other icons / icon libraries used by Red Hat (if applicable).